code/modules/mob/new_player/server_info_window.dm

```diff
@@ -0,0 +1,52 @@
+/client/verb/check_server_info()
+	set name = "Server Information"
+	set category = "OOC"
+
+	var/output = join_motd
+	var/list/close = list("Gotcha!","AFFIRMATIVE","EXCELSIOR!","I accept","The Pact Is Sealed","Continue","Take me to the fun!","Let us begin")
+	output += file2text("config/rules.html")
+	output += "<p><b>Map info:</b></p>"
+	output += using_map.motd
+	output += {"<br>
```

Comment:
If you're going to use a multi-line string, you can embed the other stuff in it with brackets. eg. 
```
{"[join_motd]
[file2text("config/rules.html")]
<p><b>Map info:</b></p>
[using_map.motd]<br>
	. . .
```
Saves a few string operations.